src/ckb/network.rs

```
@@ -1076,6 +1076,24 @@ where
         debug!("Network service for {:?} shutdown", state.peer_id);
         Ok(())
     }
+
+    async fn handle_supervisor_evt(
+        &self,
+        _myself: ActorRef<Self::Msg>,
+        message: SupervisionEvent,
+        _state: &mut Self::State,
+    ) -> Result<(), ActorProcessingErr> {
+        match message {
+            SupervisionEvent::ActorTerminated(who, _, _) => {
+                debug!("Actor {:?} terminated", who);
+            }
+            SupervisionEvent::ActorPanicked(who, _) => {
```

Better fail fast to expose the panic issue early.